packages/react-tabster/src/hooks/useTabster.ts

```diff
@@ -0,0 +1,20 @@
+import { useFluent } from '@fluentui/react-shared-contexts';
+import { getCurrentTabster, createTabster } from 'tabster';
+
+/**
+ * Tries to get a tabster instance on the current window or creates a new one
+ * @internal
+ * @returns Tabster core instance
+ */
+export const useTabster = () => {
+  const { targetDocument } = useFluent();
+
+  const defaultView = targetDocument?.defaultView || undefined;
+  const tabsterOptions = { autoRoot: {} };
+
+  if (!defaultView) {
+    return null;
+  }
+
+  return getCurrentTabster(defaultView) ?? createTabster(defaultView, tabsterOptions);
```

Comment:
Should we wrap this part with `React.useMemo()` for better performance?

Reply:
Not until we start using tabster creation using options, the `getCurrentTabster` call itself is extremely lightweight since it just tries to grab an object from window

Reply:
I'll add a TODO comment